interpreter/interpretable.go

```diff
@@ -68,6 +68,8 @@ type InterpretableAttribute interface {
 
 	// Resolve returns the value of the Attribute given the current Activation.
 	Resolve(Activation) (interface{}, error)
+
+	Cost() (min, max int64)
```

Comment:
remove? I don't think this is needed

Reply:
It is for calculating cost for the attribute. Do we wanna assume cost for const and getting attribute all 0?

Reply:
CEL currently uses https://github.com/google/cel-go/blob/a29469fda231aa37802d476ef818e2224c25e157/interpreter/coster.go#L24 instead of adding Cost to individual interfaces. Let's continue with that approach.